experimental/packages/sampler-composite/src/tracestate.ts

```diff
@@ -32,12 +32,10 @@ export type OtelTraceState = {
   rest?: string[];
 };
 
-export function invalidTraceState(): OtelTraceState {
-  return {
-    randomValue: INVALID_RANDOM_VALUE,
-    threshold: INVALID_THRESHOLD,
-  };
-}
+export const INVALID_TRACE_STATE: OtelTraceState = {
+  randomValue: INVALID_RANDOM_VALUE,
+  threshold: INVALID_THRESHOLD,
+};
```

Comment:
```suggestion
export const INVALID_TRACE_STATE: OtelTraceState = Object.freeze({
  randomValue: INVALID_RANDOM_VALUE,
  threshold: INVALID_THRESHOLD,
});
```

IIUC, using Object.freeze here should help prevent accidental mutation of this re-used object.

There is *one* current usage in opentelemetry-js.git: https://github.com/open-telemetry/opentelemetry-js/blob/f2b0d2a422636d7f171f329a14d5452951bd3771/experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/OTLPMetricExporterBase.ts#L115-L117
I only went looking because I don't have a lot of personal experience using Object.freeze.